Tip to configure antivirus on Windows (#2358)
restic/restic: doc/faq.rst

Why does restic perform so poorly on Windows?
---------------------------------------------

> In some cases the real-time protection of antivirus software can interfere with restic's operations. If you are experiencing bad performance you can try to temporarily disable your antivirus software to find out if it is the cause for your performance problems. If you find out that the antivirus is the cause, and you want to gain maximum performance, you have to add restic process (the full path) to the exclusions list.

Reviewer: ```suggestion
In some cases the real-time protection of antivirus software can interfere with restic's operations. If you are experiencing bad performance you can try to temporarily disable your antivirus software to find out if it is the cause for your performance problems. If you are certain that the antivirus software is the cause for this and you want to gain maximum performance, you have to add the restic binary to an exclusions list within the antivirus software.
```
